### PR TITLE
Update index.html

### LINKED ISSUE #1585: Improvise
**Actual Behaviour**

Image is added
**Expected Behaviour**

Change it to a gradient

**Steps to reproduce it**

Update files

**Would you like to work on the issue?**

Yes I am

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Fixes #1585 

Changes: Added a gradient replaced image
Deploy link: https://sheltered-dawn-50259.herokuapp.com/
  
  
screen shot 2018-01-07 at 9 58 18 am

### CODE REVIEW

**Comment by arjunpandey2003** [comment]
I can make it a darker shade if you want

**Comment by samyakbvs** [comment]
@arjunpandey2003 it looks good but pls get your color combinations right

**Comment by samyakbvs** [comment]
The background is perfect it would be better if you change the color of the text

**Comment by arjunpandey2003** [comment]
@samyakbvs instead i make the background a bit dark?

**Comment by geekyd** [comment]
@arjunpandey2003 Use a darker background & use white colour for text.

**Comment by arjunpandey2003** [comment]
@samyakbvs and @geekyd i have made the changes

File: src/www/index.html [hunk 1]
```diff
@@ -150,12 +150,17 @@
...
-     background-image: url(chill.jpg);
+     background: #000428; /* fallback for old browsers */
+     background: -webkit-linear-gradient(to right, #000428, #004e92); /* Chrome 10-25, Safari 5.1-6 */
+     background: linear-gradient(to right, #000428, #004e92); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
     }
     
     body:hover .overlay{
     opacity: 1;
     }
+    #white{
```
Comment (geekyd) [inline src/www/index.html]: Please remove this class we already have `white-color` class

**Comment by arjunpandey2003** [comment]
@geekyd please merge

**Comment by geekyd** [comment]
@arjunpandey2003 Please provide screenshots of fresh deployment. Some changes were made to the landing page.

**Comment by arjunpandey2003** [comment]
ok sure

**Comment by arjunpandey2003** [comment]
I am making a fresh new PR

**Pull request closed without merging**